django-backend/fecfiler/authentication/views.py

```diff
@@ -37,39 +51,136 @@ class AccountViewSet(GenericViewSet, ListModelMixin):
         "email",
         "role",
         "is_active",
-        "name"
+        "name",
     ]
     ordering = ["name"]
 
     def get_queryset(self):
-        queryset = Account.objects.annotate(
-            name=Concat('last_name', Value(', '), 'first_name', output_field=CharField())
-        ).filter(cmtee_id=self.request.user.cmtee_id).all()
+        queryset = (
+            Account.objects.annotate(
+                name=Concat(
+                    "last_name", Value(", "), "first_name", output_field=CharField()
+                )
+            )
+            .filter(cmtee_id=self.request.user.cmtee_id)
+            .all()
+        )
 
         return queryset
 
 
 class LoginDotGovSuccessSpaRedirect(View):
     def get(self, request, *args, **kwargs):
+        request.session["user_id"] = request.user.pk
         redirect = HttpResponseRedirect(LOGIN_REDIRECT_CLIENT_URL)
-        redirect.set_cookie(FFAPI_COMMITTEE_ID_COOKIE_NAME,
-                            request.user.cmtee_id,
-                            domain=FFAPI_COOKIE_DOMAIN,
-                            secure=True)
-        redirect.set_cookie(FFAPI_EMAIL_COOKIE_NAME,
-                            request.user.email,
-                            domain=FFAPI_COOKIE_DOMAIN,
-                            secure=True)
+        redirect.set_cookie(
+            FFAPI_COMMITTEE_ID_COOKIE_NAME,
+            request.user.cmtee_id,
+            domain=FFAPI_COOKIE_DOMAIN,
+            secure=True,
+        )
+        redirect.set_cookie(
+            FFAPI_EMAIL_COOKIE_NAME,
+            request.user.email,
+            domain=FFAPI_COOKIE_DOMAIN,
+            secure=True,
+        )
         return redirect
 
 
 class LoginDotGovSuccessLogoutSpaRedirect(View):
     def get(self, request, *args, **kwargs):
         response = HttpResponseRedirect(LOGIN_REDIRECT_CLIENT_URL)
-        response.delete_cookie(FFAPI_COMMITTEE_ID_COOKIE_NAME,
-                               domain=FFAPI_COOKIE_DOMAIN)
-        response.delete_cookie(FFAPI_EMAIL_COOKIE_NAME,
-                               domain=FFAPI_COOKIE_DOMAIN)
-        response.delete_cookie('csrftoken',
-                               domain=FFAPI_COOKIE_DOMAIN)
+        response.delete_cookie(
+            FFAPI_COMMITTEE_ID_COOKIE_NAME, domain=FFAPI_COOKIE_DOMAIN
+        )
+        response.delete_cookie(FFAPI_EMAIL_COOKIE_NAME, domain=FFAPI_COOKIE_DOMAIN)
+        response.delete_cookie("csrftoken", domain=FFAPI_COOKIE_DOMAIN)
         return response
+
+
+class IsAccountOwner(permissions.BasePermission):
```

Comment:
Is this used anywhere?

Reply:
This class is not used. It has been removed from the code.